aip/general/0193.md

```diff
@@ -210,41 +280,237 @@ map for the error payload to be backwards compatible, even if the value
 for a particular key is empty. Keys **must** be expressed as lower
 camel-case.
 
-#### Localization
+##### Localization
+
+If a localized error message is required, the service **must** include
+[`google.rpc.LocalizedMessage`][LocalizedMessage] in `Status.details`.
+The value of the `Status.message` string **should** be presented in the
+service's native language in the `LocalizedMessage.message` string,
+while also ensuring that the `locale` string shows the correct language.
+
+##### LocalizedMessage payload
+
+The LocalizedMessage payload **should** contain the complete resolution
+to the error. If more information is needed than can fit in this
+payload, then additional resolution information **must** be provided in
+the HelpLink payload.  See the [HelpLink payload](#helplink-payload)
+section for guidance.
+
+For pre-existing (brownfield) APIs, the content in this `message` string
+will differ from the `message` string in `google.rpc.status`.
+
+[LocalizedMessage Payload][LocalizedMessage]
+
+Details of this object are summarized in the following fields, field
+descriptions, and examples:
+
+- `locale string`
+  - The locale that follows the specification defined in [IETF
+    bcp47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt) (Tags for
+    Identifying Languages).
+  - *Example*: "en-US" , "fr-CH", "es-MX"
+- `message string`
```

Comment:
I would call out that dynamic segments--variables--in the Help.message also must be in ErrorInfo.metadata. Otherwise, users will parse it and it will break clients to change the format of the string.

Reply:
@andrei-scripniciuc 
Andrei, Agreed?
Do you mean Helplink.message?

Reply:
Done. Also, Sam's comment seems to be resolved by this sentence: "The message
    **should** include, where needed, data provided in other fields such
    as metadata."